analysis/test_cases.go

```diff
@@ -6,13 +6,13 @@ var Tier0TestCases = []TC{
 	TackleTestappPublicWithDeps,
 	TackleTestappPublicPackageFilter,
 	AcmeairWebapp,
-	Tomcat,
+	BookServerVerified,
```

Comment:
Moved Tomcat to TIER0 (little duplicate to app already in TIER0) and added BookServer to TIER0.